uint8 status
Header header
float64 probMiss
float64[20] bt
float64[20] prob
float64[2] targetPosDrone
float64[2] targetPosSegway
